interfaces/udev/backend_test.go

```diff
@@ -322,6 +323,22 @@ func (s *backendSuite) TestCombineSnippetsWithActualSnippets(c *C) {
 	}
 }
 
+func (s *backendSuite) TestControlsDeviceCgroup(c *C) {
+	// NOTE: Hand out a permanent snippet so that .rules file is generated.
+	s.Iface.UDevPermanentSlotCallback = func(spec *udev.Specification, slot *snap.SlotInfo) error {
+		spec.SetControlsDeviceCgroup()
+		spec.AddSnippet("dummy")
```

Comment:
the test in this style is probably more interesting if you switch the order of these two, first AddSnippet then Set...

Reply:
Done